Purchasing.Mvc/Views/Report/ProcessingTimeByRole.cshtml

```diff
@@ -9,9 +9,9 @@
 
 <h2>ProcessingTimeByRole</h2>
 
-<script src="http://code.highcharts.com/highcharts.js"></script>
-<script src="http://code.highcharts.com/highcharts-more.js"></script>
-<script src="http://code.highcharts.com/modules/exporting.js"></script>
+<script src="https://code.highcharts.com/highcharts.js"></script>
+<script src="https://code.highcharts.com/highcharts-more.js"></script>
+<script src="https://code.highcharts.com/modules/exporting.js"></script>
```

Comment:
@jSylvestre stick a /5/ after the com, like https://code.highcharts.com/5/modules/exporting.js